.github/workflows/publish-draft-release.yml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   publish-draft-release:
-    runs-on: ubuntu-22.04
+    runs-on: self-hosted
```

Comment:
Currently the published release does not contain the built binaries, but it is only a tag. With that said I think this can stay on the default GHA Runners as well, because it does not source code files either.

Reply:
ok